### PR TITLE
adds bricklayers mallet and obtainment

### PR DESCRIPTION
**Pull request by SkippZzMods**
Adds the bricklayers mallet accessory and obtainment

https://trello.com/c/AdpaxwmA/249-bricklayers-mallet-45 - follows it perfectly

not that I can think of

### CODE REVIEW

File: Content/Items/Misc/Accessories.BricklayersMallet.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+        public override void SafeUpdateEquip(Player Player)
+        {
+            Player.blockRange *= 2;
+            if (Main.myPlayer == Player.whoAmI)
+            {
+                Player.tileRangeX *= 2;
+                Player.tileRangeY *= 2;
+
+                bool outsideXRange = Main.MouseWorld.X < Player.Center.X ? Math.Abs(Player.Left.X - Main.MouseWorld.X) > 80f : Math.Abs(Player.Right.X - Main.MouseWorld.X) > 80f;
```
Comment (ScalarVector1) [inline Content/Items/Misc/Accessories.BricklayersMallet.cs:31]: Should these be hardcoded? What about other vanilla accessories that extend the default range?

**Review by ScalarVector1 — COMMENTED** [review]
Looks good for the most part, slightly concerned about hardcoding the range for the slowdown though.

File: Content/Items/Misc/Accessories.BricklayersMallet.cs (hunk 1 above)
Comment (SkippZzMods) [reply Content/Items/Misc/Accessories.BricklayersMallet.cs:31]: your default range still stays the same? if something increases your default, that isn't your new default

Comment (ScalarVector1) [reply Content/Items/Misc/Accessories.BricklayersMallet.cs:31]: default meaning "not affected by this accessory" here. This is currently written to override all additive effects that apply before this.

**Pull request merged**